source/codegen/metadata/nidigitalpattern/config.py

```diff
@@ -1,31 +1,16 @@
-﻿# -*- coding: utf-8 -*-
-# This file is generated from NI-Digital Pattern Driver API metadata version 19.5.0d7
+# -*- coding: utf-8 -*-
+# This file is generated from NI-Digital Pattern Driver API metadata version 22.8.0d9999
 config = {
     'additional_headers': {
-        'custom/ivi_errors.h': [
```

Comment:
Can you add this back? In hapigen, you can see [here in Azure](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/357617?_a=files&path=/src/nifgen/niFgenBaseDistribution/fgenAPIMetadata/source/fgenAPIMetadata/grpc_device/global_config/addition.hapigen) for an example